metadata/base/metadata-ui-deployment.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: ui
+  labels:
+    app: metadata-ui
+spec:
+  selector:
+    matchLabels:
+      app: metadata-ui
+  template:
+    metadata:
+      name: ui
+      labels:
+        app: metadata-ui
+    spec:
+      containers:
+      # TODO(rjbauer): Change this to something like gcr.io/metadata/frontend:0.0.1
+      - image: gcr.io/kubeflow-images-public/metadata-frontend:v0.0.1
```

Comment:
`v0.1.8` is the most recent image with the filtering and sorting support. Can we update this to point to that image and remove the TODO. Alternatively, I can remove the other versions and make the latest image `v0.1.7` to match the backend service although I don't anticipate they will still in sync for long.